index.html

```diff
@@ -555,7 +555,7 @@ <h3 id="profile-description-methodology">Methodology</h3>
         other or overlap, extended profiles can be built on top
         of the core profile.</p>
       <p>
-        This specification does not put any requirements on the 
+        This specification does not put any requirements on the
         scope and contents of other profiles.
```

Comment:
I'm not sure if the first part of the sentence is correct. Instead "This specification does ..." it should be more "A specified profile does not put any requirements on the scope and contents of other profiles."